alerter.c

```diff
@@ -23,10 +38,19 @@ void alertInCelcius(float farenheit) {
     }
 }
 
-int main() {
-    alertInCelcius(400.5);
+
+//Main function
+int main(void){
+     alertInCelcius(400.5);

```

Comment:
Atleast 3 test values need to be covered i.e. below threshold, threshold and above threshold. If threshold is 392 F (i.e. 200 c) then consider values <392, =392 and >392

Reply:
Updated review comments